index.js

```diff
@@ -39,6 +39,9 @@ export default class MaterialButton extends PureComponent {
   renderLoadingAnim() {
     if (this.props.isLoading) {
       let color = this.props.flat ? this.props.color : '#ffffff';
+      if (this.props.hasOwnProperty('textStyle') && this.props.textStyle.hasOwnProperty('color')) {
```

Comment:
`textStyle` may not always be a plain object. Please consider `StyleSheet` as well. Or take `textColor` as a prop instead,